src/refract/SerializeVisitor.cc

```diff
@@ -22,6 +23,17 @@ namespace refract
             for (iterator it = collection.begin(); it != collection.end(); ++it) {
                 SerializeCompactVisitor s;
                 s.visit(*(*it));
+                
```

Comment:
just some console logs so I can identify the structure, this should be removed from pull request